React/Getting started with React.md

```diff
@@ -0,0 +1,33 @@
+# Getting started with React
+
+### React
```

Comment:
I think these headings should be larger, so `##` instead of `###`. The same goes for all of the other heading in this PR.